content/docs/render-props.md

```diff
@@ -42,19 +42,19 @@ class MouseTracker extends React.Component {
   render() {
     return (
       <div style={{ height: '100%' }} onMouseMove={this.handleMouseMove}>
-        <h1>Move the mouse around!</h1>
-        <p>The current mouse position is ({this.state.x}, {this.state.y})</p>
+        <h1>Gerakkan tetikus!</h1>
+        <p>Posisi tetikus saat ini adalah ({this.state.x}, {this.state.y})</p>
       </div>
     );
   }
 }
 ```
 
-As the cursor moves around the screen, the component displays its (x, y) coordinates in a `<p>`.
+Ketika kursor  bergerak di layar, komponen menampilkan koordinat (x, y) dari kursor di sebuah `<p>`.
 
-Now the question is: How can we reuse this behavior in another component? In other words, if another component needs to know about the cursor position, can we encapsulate that behavior so that we can easily share it with that component?
+Kemudian muncul pertanyaan: Bagaimana kita bisa menggunakan kembali perilaku ini pada komponen lainnya? Dengan kata lain, apabila ada komponen lain yang membutuhkan informasi tentang posisi kursor, dapatkah kita merangkum perilaku ini sehingga kita dapat dengan mudah membagikan informasi posisi kursor kepada komponen tersebut?
```

Comment:
Indeed. I only used it because that's what google translates suggested. It doesn't sound right somehow. I've posted in #2 regarding this but haven't got any comments. Do you have any suggestion? Would _mengenkapsulasi_ be better instead?

Reply:
@imiro I guess `mengenkapsulasi` does sounds better.